pkg/runtime/BUILD.bazel

```diff
@@ -3,10 +3,16 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 go_library(
     name = "go_default_library",
     srcs = [
+        "controller.go",
```

Comment:
I agree, nothing for this PR. However, after 0.2, may be we can move the control loop related codebase into a separate pkg..